reference/src/index-syntax.md

```diff
@@ -20,13 +20,13 @@ module matrix {
     public struct Matrix<T> { v: vector<vector<T>> }
 
     #[syntax(index)]
-    public fun borrow<T>(s: &Matrix<T>, i: u64, j: u64):  &T {
-        borrow(borrow(s.v, i), j)
+    public fun borrow<T>(s: &Matrix<T>, i: u64, j: u64): &T {
+        vector::borrow(vector::borrow(&s.v, i), j)
     }
 
     #[syntax(index)]
-    public fun borrow_mut<T>(s: &mut Matrix<T>, i: u64, j: u64):  &mut T {
-        borrow_mut(borrow_mut(s.v, i), j)
+    public fun borrow_mut<T>(s: &mut Matrix<T>, i: u64, j: u64): &mut T {
+        vector::borrow_mut(vector::borrow_mut(&mut s.v, i), j)
```

Comment:
```suggestion
        &mut s.v[i][j]
```

Reply:
Hmmmm, maybe? I was avoiding having to use index syntax to explain index syntax. Thoughts?